src/components/Home/HomeComponent.js

```diff
@@ -23,28 +44,111 @@ type Props = {
  */
 
 export default class HomeComponent extends Component<Props> {
+  constructor(props) {
+    super(props);
+    this.styleOver = this.styleOver.bind(this);
+    this.styleNotOver = this.styleNotOver.bind(this);
+    this.skipVideo = this.skipVideo.bind(this);
+  }
+
+  state = {
+    displayVideo: true
+  }
+
+  componentWillMount(): void {
+    this.styleNotOver();
+  }
+
+  componentDidMount() {
+    // subscribe state change
+    // eslint-disable-next-line
+    this.refs.player.subscribeToStateChange(this.handleStateChange.bind(this));
```

Comment:
Je vais voir pourquoi les tests ne passent pas ici 👍

Reply:
Essaye de faire un truc comme ça : 
- `this.player= React.createRef();` dans ton `constructor` 
- Tu peux donc faire `ref={this.player}` dans ton composant
- Changer `this.refs.player` dans ton ComponentDidMount en `this.player`